### PR TITLE
Parameterised docker image for DBs

### PR DESCRIPTION
**Pull request by rsvoboda**
Parameterised docker image for DBs

### CODE REVIEW

**Review by geoand — APPROVED** [review]
LGTM

File: integration-tests/jpa-postgresql/pom.xml [hunk 1]
```diff
@@ -169,7 +174,7 @@
                         <configuration>
                             <images>
                                 <image>
-                                    <name>postgres:10.5</name>
+                                    <name>${postgresql.image}</name>
```
Comment (gsmet) [inline integration-tests/jpa-postgresql/pom.xml]: It should be consistent with `postgres.url` a few lines above. So either we fix the existing one (and the CI jobs) or we stick to `postgres` as the prefix.

**Review by gsmet — DISMISSED** [review]

File: integration-tests/jpa-postgresql/pom.xml (hunk 1 above)
Comment (rsvoboda) [reply integration-tests/jpa-postgresql/pom.xml]: Updated and pulled up the url property so all 4 modules are in sync

**Comment by rsvoboda** [comment]
@gsmet PR updated, but CI seems to be having hard time

**Comment by geoand** [comment]
Yeah, I saw that in other PRs as well...

**Comment by rsvoboda** [comment]
@gsmet added triage/backport-1.3? label, this is needed for checks against different versions of databases

**Pull request merged**